supporting_content/appendix/quantity_driven_pricing_strategies.md

```diff
@@ -45,26 +45,27 @@ For better comprehension, please refer to the sample price-tiers configuration a
 
 #### Sample Billing data
 
-| Scenario | QuantityInPricingUnit | PricingUnit | ListUnitPrice | NegotiatedUnitPrice | BilledUnitPrice | PricingCurrency | BillingCurrency | ListCost | BilledCost | EffectiveCost |
-| :------- | --------------------: | :---------- | ------------: | ------------------: | --------------: | :-------------- | :-------------- | -------: | ---------: | ------------: |
-| S-1      |                    10 | 1GB         |             1 |                0.90 |            0.90 | USD             | USD             |       10 |          9 |             9 |
-| S-1      |                     2 | 1GB         |          0.50 |                0.45 |            0.45 | USD             | USD             |        1 |        0.9 |           0.9 |
-| S-2      |                    12 | 1GB         |          0.50 |                0.45 |            0.45 | USD             | USD             |        6 |        5.4 |           5.4 |
+| Scenario | PricingQuantity | PricingUnit | ListUnitPrice | NegotiatedUnitPrice (1.0+) | PricingCurrency (1.0+) | BillingCurrency | ListCost | BilledCost | EffectiveCost |
```

Comment:
As a FOCUS contributor, I know that `(1.0+)` means "these columns, although important to this example, aren't making the FOCUS 1.0 spec" -- but as a general audience member, these parentheticals without any further context are confusing to me.  Some verbiage above the table, even a single sentence, that explains these would be helpful.